helper/namespace/namespace.go

```diff
@@ -26,6 +26,7 @@ type contextValues struct{}
 
 type Namespace struct {
 	ID             string            `json:"id" mapstructure:"id"`
+	Parent         string            `json:"parent" mapstructure:"parent"`
```

Comment:
I don't like that this is part of Namespace and not NamespaceEntry. The UUID itself is part of the entry.